front/app/components/EsriMap/index.tsx

```diff
@@ -98,144 +80,122 @@ const EsriMap = ({
   const isMobileOrSmaller = useBreakpoint('phone');
   const { data: appConfig } = useAppConfiguration();
 
-  const [map, setMap] = useState<Map | null>(null);
-  const [webMap, setWebMap] = useState<WebMap | null>(null);
+  const [map, setMap] = useState<Map | WebMap | null>(null);
```

Comment:
I found it a bit confusing that we have multiple map variables even though we only have one map at a given time (either a normal `Map` or a `WebMap`). So I put them into one variable